.github/workflows/pre-release.yml

```diff
@@ -1,14 +1,19 @@
 name: Release 2 - Pre-release testing
-run-name: Pre-release Testing of ${{ github.event.workflow_run.name }}
+run-name: Publish Release ${{ inputs.version }}
 
 on:
-  workflow_run:
-    workflows: [Release 1 - Build Release Artifact]
-    types: [completed]
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Metabase version (e.g. v0.46.3)'
+        type: string
+        required: true
+      commit:
+        description: 'A full-length commit SHA-1 hash'
+        required: true
 
 jobs:
   release-artifact:
```

Comment:
This was me shooting myself in the foot.
It took me good five minutes to figure out why is this "A release artifact" and not "Release the artifact". We should change the name to download release artifact or obtain the release artifact.

IIRC, you even suggested something like this a few months ago.
Sorry for not implementing it back then.